weave-js/src/components/Panel2/ChildPanelExportReport/ReportSelection.tsx

```diff
@@ -0,0 +1,151 @@
+import * as w from '@wandb/weave/core';
+import React from 'react';
+
+import {useNodeValue} from '../../../react';
+import {Select} from '../../Form/Select';
+import {Icon} from '../../Icon';
+import {ChildPanelFullConfig} from '../ChildPanel';
+import {
+  customEntitySelectComps,
+  customReportSelectComps,
+} from './customSelectComponents';
+import {EntityOption, ReportOption, useEntityAndProject} from './utils';
+import {Button} from '../../Button';
+
+type ReportSelectionProps = {
+  config: ChildPanelFullConfig;
+  selectedEntity: EntityOption | null;
+  selectedReport: ReportOption | null;
+  setSelectedEntity: (entity: EntityOption) => void;
+  setSelectedReport: (report: ReportOption | null) => void;
+};
+
+export const ReportSelection = ({
+  config,
+  selectedEntity,
+  selectedReport,
+  setSelectedEntity,
+  setSelectedReport,
+}: ReportSelectionProps) => {
+  const {entityName} = useEntityAndProject(config);
+
+  // Get all of user's entities
+  const entitiesMetaNode = w.opMap({
+    arr: w.opUserEntities({user: w.opRootViewer({})}),
+    mapFn: w.constFunction({row: 'entity'}, ({row}) => {
+      return w.opDict({
+        name: w.opEntityName({entity: row}),
+        isTeam: w.opEntityIsTeam({entity: row}),
+      } as any);
+    }),
+  });
+  const entities = useNodeValue(entitiesMetaNode);
+
+  // Get list of reports across all entities and projects
+  const reportsNode = w.opEntityReports({
+    enity: w.opRootEntity({
```

Comment:
**typo:** 
```suggestion
    entity: w.opRootEntity({
```

Reply:
What's fascinating is that it didn't cause errors until I was working on adding the project selector...but fixed!